RELEASE_NOTES.md

```diff
@@ -84,7 +84,7 @@ This release replaces the `@actor` decorator with a new `Actor` class.
 
 ## New Features
 
-- DFS for compentent graph
+- Addded `DFS` to the compentent graph
```

Comment:
this is adding more typos :D

Reply:
s/compentent/component
s/Addded/Added

Reply:
Just wanted to check if you are still focused!